projboard/forms.py

```diff
@@ -9,3 +9,8 @@ class Meta:
         widgets = {
             'user_id': forms.HiddenInput(),
         }
+
+
+class LoginForm(forms.Form):
+    username = forms.CharField(label='Username', max_length=100)
+    password = forms.CharField(widget=forms.PasswordInput)
```

Comment:
In my opinion we should match the max_length of password here with the max_length of password in User model.
Take a look [here](https://github.com/redhat-beyond/Smarticle/blob/main/projboard/models/user.py#L13)

Reply:
In addition , why you aren't using this form in the html page ?

Reply:
done